### DISCUSSION TITLE
Kubernetes Ingress support in self-hosted gateway - is it still experimental?

### DISCUSSION THREAD

**Discussion by richorama** [Azure/api-management-self-hosted-gateway-ingress#14] [Q&A] [answered]
Hi, would you be able to provide any information on the K8s ingress support on the self-hosted gateway?

Is it still experimental? Is it likely to become suitable for production workloads?

Many thanks,

Richard. (rastbury@microsoft.com)

**Comment by ScratZa** [comment]
Also interested - is this still experimental ?

**Comment by tomkerkhove** [comment, marked as answer]
Yes, this is currently still experimental.

(sorry for the slow response)
